### PR TITLE
Add Operations to Local Testnet install document

### PR DESCRIPTION
**Pull request by BenjaminMoe**
This Pull Request adds documentation and screenshots for creating a wallet, creating, and then resolving a did to confirm that the node install has been completed successfully. 

Document can be seen with images here:
https://github.com/transmute-industries/sidetree.js/blob/docs/local-testnet/packages/did-method-element/docs/local-test.md

### CODE REVIEW

File: packages/did-method-element/docs/local-test.md [hunk 1]
```diff
@@ -150,7 +152,7 @@ to copy the address to the clickboard.
 
 ![Screenshot from 2021-12-09 03-19-46](https://user-images.githubusercontent.com/86194145/145262382-c708ab4c-c020-462b-84f9-87ce75ef76b7.png)
 
-From there we want to goto the [Ropsten Etereum Faucet](https://faucet.ropsten.be/),
+From there we want to goto the [Ropsten Etereum Faucet](https://faucet.ropsten.be/) (recomend using ingonito tab),
```
Comment (OR13) [inline packages/did-method-element/docs/local-test.md:155]:
```suggestion
From there we want to goto the [Ropsten Ethereum Faucet](https://faucet.ropsten.be/) (recomend using ingonito tab),
```

File: packages/did-method-element/docs/local-test.md [hunk 2]
```diff
@@ -184,55 +191,74 @@ https://ropsten.infura.io/v3/[YOUR_PROJECT_API_KEY]
...
-$ npm run start
-```
+Once we have Element running on our local machine, we can then run a test
+operation to see if we are able to create and then resolve a did. 
+
+![Screenshot from 2021-12-14 22-46-41](https://user-images.githubusercontent.com/86194145/146037402-ae75eb83-0394-40f2-bd0e-ecfc090aed29.png)
+
+We can start by opening up `localhost:3000` in our browser and then clicking on the `Wallet` link on the "Manage" card. 
+
+![Screenshot from 2021-12-14 22-47-22](https://user-images.githubusercontent.com/86194145/146037619-c62917e7-a387-40c5-8207-2e8123302627.png)
```
Comment (OR13) [inline packages/did-method-element/docs/local-test.md:238]: consider renaming screenshots

File: packages/did-method-element/docs/local-test.md [hunk 3]
```diff
@@ -18,8 +18,10 @@ set up and run and Ethereum node locally.
 
 - Install Dependencies
 - Clone and Build the Repository
-- Start Required Services
+- Create and Fund a Wallet
```
Comment (OR13) [inline packages/did-method-element/docs/local-test.md:21]: consider a better name for this file than "local-test.md"

File: packages/did-method-element/docs/local-test.md (hunk 2 above)
Comment (BenjaminMoe) [reply packages/did-method-element/docs/local-test.md:238]: Sure, I'll change the alt text to describe the operation.

File: packages/did-method-element/docs/local-test.md (hunk 1 above)
Comment (BenjaminMoe) [reply packages/did-method-element/docs/local-test.md:155]: fix spelling mistake.

File: packages/did-method-element/docs/local-test.md (hunk 3 above)
Comment (BenjaminMoe) [reply packages/did-method-element/docs/local-test.md:21]: `local-element-ropsten-install.md`?

**Review by OR13 — APPROVED** [review]

**Pull request merged**